plugins/browser-plugin-web-vitals/src/types.ts

```diff
@@ -0,0 +1,29 @@
+export interface ReportCallback {
+  (metric: Metric): void;
+}
+
+/**
+ * @interface Metric
+ * Partial metric interface from https://github.com/GoogleChrome/web-vitals
+ */
+interface Metric {
```

Comment:
Should we import this type from `web-vitals` so a change has the chance to break our build? Or is the build-dep too heavy to be worth it?

Reply:
Added it and confirmed that since we are using only type imports, the final bundle is not affected.